test/Microsoft.TemplateEngine.Cli.UnitTests/ParserTests/Approvals/TabCompletionTests.RootCommand_GetAllSuggestions.verified.txt

```diff
@@ -62,41 +62,6 @@
     InsertText: webconfig,
     Documentation: A file used to configure Web Application settings
   },
-  {
```

Comment:
check this

Reply:
I think there was a bug around this that got fixed? https://github.com/dotnet/command-line-api/issues/1563 but I would expect that fix to have been picked up in this repo already..

Reply:
this was happening due to help was not enabled when parsing, fixed it.